src/constraint_system/composer.rs

```diff
@@ -737,20 +715,14 @@ impl TurboComposer {
     /// the largest amount of performance and the minimum circuit-size
     /// possible. Since it allows the end-user to set every selector coefficient
     /// as scaling value on the gate eq.
-    pub fn plonkup_gate(
+    pub fn append_plonkup_gate(
```

Comment:
We have simple raw gates as `append_plonkup_gate` and `append_gate`, and complex ones as `gate_boolean`.

I agree there is some ambiguity here. What if we rename from `append_plonkup_gate/append_gate` to `append_plonkup_constraint/append_constraint`?

Reply:
The name "constraint" sounds better to me. Would still like plonkup to be represented in gate-space, but I think this will do for now. @ZER0 thoughts?